src/components/pod/pod.test.js

```diff
@@ -13,6 +55,234 @@ const PodWithSoftDelete = () => (
 );
 
 context("Testing Pod component", () => {
+  describe("should render Pod component", () => {
+    it.each([
+      [true, "0px none rgb(0, 0, 0)"],
+      [false, "0px none rgb(0, 0, 0)"],
+    ])(
+      "should check when border is %s for Pod component",
+      (boolVal, border) => {
+        CypressMountWithProviders(<PodComponent border={boolVal} />);
+        podComponent().should("have.css", "border", border);
+      }
+    );
+
+    it.each(specialCharacters)(
+      "should check className as %s for Pod component",
+      (className) => {
+        CypressMountWithProviders(<PodComponent className={className} />);
+        podComponent().should("have.class", className);
+      }
+    );
+
+    it.each([
+      ["extra-small", "121px", "61.5625px"],
+      ["small", "121px", "61.5625px"],
+      ["medium", "137px", "77.5625px"],
+      ["large", "185px", "93.5625px"],
+      ["extra-large", "217px", "125.5625px"],
+    ])("should check %s size for Pod component", (size, height, width) => {
+      CypressMountWithProviders(<PodComponent size={size} />);
+      divPodComponent()
+        .should("have.css", "height", height)
+        .and("have.css", "width", width);
+    });
+
+    it.each([
+      [
+        "primary",
+        "rgb(255, 255, 255) none repeat scroll 0% 0% / auto padding-box border-box",
+        "none",
+      ],
+      [
+        "secondary",
+        "rgb(242, 245, 246) none repeat scroll 0% 0% / auto padding-box border-box",
+        "none",
+      ],
+      [
+        "tertiary",
+        "rgb(237, 241, 242) none repeat scroll 0% 0% / auto padding-box border-box",
+        "none",
+      ],
+      [
+        "tile",
+        "rgb(255, 255, 255) none repeat scroll 0% 0% / auto padding-box border-box",
+        "rgba(2, 18, 36, 0.2) 0px 2px 3px 0px",
+      ],
+      [
+        "transparent",
+        "rgba(0, 0, 0, 0) none repeat scroll 0% 0% / auto padding-box border-box",
+        "none",
+      ],
+    ])(
+      "should check %s variant for Pod component",
```

Comment:
similar as above, something like "should check for %s variant pod, the color is %s and boxShadow is %s"